src/main/java/frc/robot/constants/KrakenSwerveConstants.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.constants;
+
+public class KrakenSwerveConstants {
+  private KrakenSwerveConstants() {
+    throw new IllegalStateException("Constants Class");
+  }
+
+  // Gear Conversions
+  public static final double TRANS_GEAR_RATIO_ROT = 1 / 6.75;
```

Comment:
you are lucky that the ratio is not int 1/6 will be 0, 1 /6.75 is double. replace with 1d / 6.75